resources/js/features/games/components/GameSetRequests/GameSetRequests.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from 'react';
+import { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { GameAvatar } from '@/common/components/GameAvatar';
+import { usePageProps } from '@/common/hooks/usePageProps';
+import { PatreonSupporterItem } from '@/features/patreon-supporters/components/PatreonSupporterItem';
+
+export const GameSetRequests: FC = () => {
+  const { game, deferredRequestors, initialRequestors, totalCount } =
+    usePageProps<App.Community.Data.GameSetRequestsPageProps>();
+  const { t } = useTranslation();
+
+  // Combine initial and deferred requestors once deferred props finish loading.
+  const allRequestors = useMemo(() => {
+    const requestors = [...initialRequestors];
+    if (deferredRequestors) {
+      requestors.push(...deferredRequestors);
+    }
+
+    return requestors;
+  }, [initialRequestors, deferredRequestors]);
+
+  return (
+    <div className="flex flex-col gap-5">
+      <h1>{'List of Set Requests'}</h1>
+
+      <div>
+        <GameAvatar {...game} size={96} />
+      </div>
```

Comment:
Consistency: let's leverage `<GameBreadcrumbs />` and `<GameHeading />` here.

_GameModificationCommentsMainRoot.tsx_ should have a good example.